apps/_scaffold/static/js/utils.js

```diff
@@ -245,10 +245,13 @@ utils.handle_flash = function() {
         return function (event) { 
             var id = 'notification-{0}'.format([element.dataset.counter]);
             element.dataset.counter = parseInt(element.dataset.counter) + 1;
+            // event.detail can be a string, but we need hash like {message: ..., class: ...}
+            var e_detail = event.detail; // event.detail is readonly!
+            if (typeof e_detail == "string"){e_detail = {message: e_detail}}
```

Comment:
I may be ok with this

Reply:
Yes, it looks not perfect, but `utils.flash([[=XML("hello world")]])` turns into `utils.flash(hello world)`, that causes an js-error